zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -299,7 +299,7 @@ <h3 class="interpreter-title">{{setting.name}}
             <td>
               <textarea ng-if="valueform.$visible" ng-model="dep.exclusions"
                         placeholder="(Optional) comma separated groupId:artifactId list"
-                        form="valueform"
+                        e-form="valueform"
```

Comment:
No `e-form` is needed here,
Actually the `e-msd-elastic` below should also be `msd-elastic`
those e-properties are only necessary in elements like `editable-text`